kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.kodilla.spring.portfolio;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class BoardConfig {
+
+    @Autowired
+    @Qualifier("taskToDoList")
+    TaskList toDoList;
```

Comment:
do we really need these instance variables?

Reply:
There's no need to use it. I changed class BoardConfig.